### ISSUE TITLE
SvgProps type is not exported in 12.4.1

### ISSUE THREAD

**Issue by zolbooo** [react-native-svg/react-native-svg#1812]
# Bug

`SvgProps` type is not exported anymore in new version of library. This breaks some Typescript builds which rely on this type.

## Environment info

Library version: 12.4.1

## Steps To Reproduce

Import `SvgProps` type from `react-native-svg`

Describe what you expected to happen:

1. SvgProps should be exported, as it was in 12.4.0

## Short, Self Contained, Correct (Compilable), Example

```ts
import Svg, {Path, SvgProps} from 'react-native-svg';
```

UPD: Here is a patch for `patch-package`
```
diff --git a/node_modules/react-native-svg/lib/typescript/ReactNativeSVG.d.ts b/node_modules/react-native-svg/lib/typescript/ReactNativeSVG.d.ts
index 9843f95..16816c4 100644
--- a/node_modules/react-native-svg/lib/typescript/ReactNativeSVG.d.ts
+++ b/node_modules/react-native-svg/lib/typescript/ReactNativeSVG.d.ts
@@ -5,7 +5,7 @@ import Ellipse from './elements/Ellipse';
 import Polygon from './elements/Polygon';
 import Polyline from './elements/Polyline';
 import Line from './elements/Line';
-import Svg from './elements/Svg';
+import Svg, {SvgProps} from './elements/Svg';
 import Path from './elements/Path';
 import G from './elements/G';
 import Text from './elements/Text';
@@ -27,5 +27,5 @@ import { parse, SvgAst, SvgFromUri, SvgFromXml, SvgUri, SvgXml } from './xml';
 import { SvgCss, SvgCssUri, SvgWithCss, SvgWithCssUri, inlineStyles } from './css';
 import { LocalSvg, WithLocalSvg, loadLocalRawResource } from './LocalSvg';
 import { RNSVGCircle, RNSVGClipPath, RNSVGDefs, RNSVGEllipse, RNSVGForeignObject, RNSVGGroup, RNSVGImage, RNSVGLine, RNSVGLinearGradient, RNSVGMarker, RNSVGMask, RNSVGPath, RNSVGPattern, RNSVGRadialGradient, RNSVGRect, RNSVGSvg, RNSVGSymbol, RNSVGText, RNSVGTextPath, RNSVGTSpan, RNSVGUse } from './elements/NativeComponents';
-export { Svg, Circle, Ellipse, G, Text, TSpan, TextPath, Path, Polygon, Polyline, Line, Rect, Use, Image, Symbol, Defs, LinearGradient, RadialGradient, Stop, ClipPath, Pattern, Mask, Marker, ForeignObject, parse, SvgAst, SvgFromUri, SvgFromXml, SvgUri, SvgXml, SvgCss, SvgCssUri, SvgWithCss, SvgWithCssUri, inlineStyles, LocalSvg, WithLocalSvg, loadLocalRawResource, Shape, RNSVGMarker, RNSVGMask, RNSVGPattern, RNSVGClipPath, RNSVGRadialGradient, RNSVGLinearGradient, RNSVGDefs, RNSVGSymbol, RNSVGImage, RNSVGUse, RNSVGTextPath, RNSVGTSpan, RNSVGText, RNSVGGroup, RNSVGPath, RNSVGLine, RNSVGEllipse, RNSVGCircle, RNSVGRect, RNSVGSvg, RNSVGForeignObject, };
+export { Svg, SvgProps, Circle, Ellipse, G, Text, TSpan, TextPath, Path, Polygon, Polyline, Line, Rect, Use, Image, Symbol, Defs, LinearGradient, RadialGradient, Stop, ClipPath, Pattern, Mask, Marker, ForeignObject, parse, SvgAst, SvgFromUri, SvgFromXml, SvgUri, SvgXml, SvgCss, SvgCssUri, SvgWithCss, SvgWithCssUri, inlineStyles, LocalSvg, WithLocalSvg, loadLocalRawResource, Shape, RNSVGMarker, RNSVGMask, RNSVGPattern, RNSVGClipPath, RNSVGRadialGradient, RNSVGLinearGradient, RNSVGDefs, RNSVGSymbol, RNSVGImage, RNSVGUse, RNSVGTextPath, RNSVGTSpan, RNSVGText, RNSVGGroup, RNSVGPath, RNSVGLine, RNSVGEllipse, RNSVGCircle, RNSVGRect, RNSVGSvg, RNSVGForeignObject, };
 export default Svg;
```

**Comment by horne3754sg** [comment]
Its not just `SvgProps`, its `Color` as well.

**Comment by WoLewicki** [comment]
@zolbooo @horne3754sg can you check if it is fixed in v12.4.2?

**Comment by horne3754sg** [comment]
Yeh its fixed in 12.4.2, cheers.
